docs/source/config_files.rst

```diff
@@ -30,6 +55,14 @@ You can also install files from pip in your ``environment.yml`` as well.
 Our example `environment.yml <https://github.com/binder-examples/python-conda_pip/blob/master/environment.yml>`_
 shows how one can specify a conda environment for repo2docker.
 
+You can also specify which Python version you are using with
+``environment.yml``. By default, ``repo2docker`` will assume you are using
+Python 3.6 with your ``environment.yml`` unless you include the version of
+Python in the file.  ``conda`` supports Python versions 2.7, 3.4, 3.5, and 3.6.
+``repo2docker`` support is best with Python 2.7, 3.5, and 3.6. If you include
```

Comment:
This line makes me think you are talking about the version of Python required to **run** `repo2docker` itself not about what is supported in terms of environments inside the container.

Reply:
> You can also specify which Python version to install in your built environment with ``environment.yml``. By default, ``repo2docker`` **installs Python 3.6** with your ``environment.yml`` unless you include the version of
> Python in the file.

Reply:
"`repo2docker` support is best with Python 2.7, 3.5, and 3.6" is what made me think this was about what version of Python is needed to run r2d. Why does r2d care which version of Python a user installs inside the container?